examples/xgboost/utils/baseline_centralized.py

```diff
@@ -20,84 +21,99 @@
 from sklearn.metrics import roc_auc_score
 from torch.utils.tensorboard import SummaryWriter
 
-# Specify training params
-data_path = "../dataset/HIGGS_UCI.csv"
-model_path_root = "../workspaces/centralized"
-round_num = 100
+parser = argparse.ArgumentParser(description="Centralized XGBoost training with random forest options")
+parser.add_argument("--num_parallel_tree", type=int, default=1, help="num_parallel_tree for random forest setting")
+parser.add_argument("--subsample", type=float, default=1, help="subsample for random forest setting")
 
-# Set mode file paths
-model_path = model_path_root + "/model_centeralized.json"
-# Set tensorboard output
-writer = SummaryWriter(model_path_root)
 
-# Load data
-start = time.time()
-higgs = pd.read_csv(data_path, header=None)
-print(higgs.info())
-print(higgs.head())
-total_data_num = higgs.shape[0]
-valid_num = 1000000
-print(f"Total data count: {total_data_num}")
-# split to feature and label
-X_higgs = higgs.iloc[:, 1:]
-y_higgs = higgs.iloc[:, 0]
-print(y_higgs.value_counts())
-end = time.time()
-lapse_time = end - start
-print(f"Data loading time: {lapse_time}")
+def main():
+    args = parser.parse_args()
+    # Specify training params
+    model_name = "centralized_" + str(args.num_parallel_tree) + "_" + str(args.subsample)
+    data_path = "./dataset/HIGGS_UCI.csv"
```

Comment:
suggest making data_path to be passed in as an argument.

Reply:
agreed